src/main/java/com/cinema/model/dto/mapper/TicketDtoMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.cinema.model.dto.mapper;
+
+import com.cinema.model.Ticket;
+import com.cinema.model.dto.TicketResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TicketDtoMapper {
+    private final MovieSessionDtoMapper movieSessionDtoMapper;
+
+    public TicketDtoMapper(MovieSessionDtoMapper movieSessionDtoMapper) {
+        this.movieSessionDtoMapper = movieSessionDtoMapper;
+    }
+
+    public TicketResponseDto mapTicketToDto(Ticket ticket) {
+        TicketResponseDto ticketResponseDto = new TicketResponseDto();
+        ticketResponseDto.setId(ticket.getId());
+        ticketResponseDto.setSession(movieSessionDtoMapper.mapToSession(ticket.getMovieSession()));
```

Comment:
```suggestion
        ticketResponseDto.setSession(movieSessionDtoMapper.mapToDto(ticket.getMovieSession()));
```
it is just for you to notice - above I suggested to remove this from here